src/constants/boxShadows/test.ts

```diff
@@ -0,0 +1,12 @@
+import compareObjectsKeysLength from 'src/utils/compareObjectsKeysLength';
+
+import PRIMARY_BOX_SHADOWS from './primary';
+import SECONDARY_BOX_SHADOWS from './secondary';
+
+describe('theme box shadows', () => {
```

Comment:
@michaeljaltamirano I noticed we have the test in the overarching theme folder. Do we need to delete this and the test in colors? or we are keeping them for granularity?

Reply:
We can remove the individual tests since the recursive nature of `compareObjectsKeysLength` captures anything in the theme.

Reply:
I removed the fonts,colors,boxShadows test. 
Tested failing them and works